### ISSUE TITLE
Allow sending BTC to oneself (for CPFP when stuck in mempool)

### ISSUE THREAD

**Issue by Jpja** [CounterpartyXCP/counterwallet#832]
CW does not allow an address to make a pure BTC send to itself.

With users having more control over fees, there will inevitably come support tickets where transactions are stuck in mempool due to too low fees. The easiest solution is to make a pure BTC transaction to oneself with a higher fee (aka Child Pays For Parent).

Sending to oneself is the most efficient as it only takes one output.

![image](https://user-images.githubusercontent.com/8447505/30850380-082bcebe-a2ae-11e7-9918-a8c7ffeb1005.png)

**Comment by deweller** [comment]
I don't believe counterwallet spends unconfirmed outputs, so the child-pays-for-parent benefit wouldn't pply here.

To really solve the stuck transactions we would need a more complex solution.

**Comment by Jpja** [comment]
Just tested it. CW does spend unconfirmed outputs.

![image](https://user-images.githubusercontent.com/8447505/31064521-a4bcc65c-a745-11e7-89a7-702f41f609dc.png)

![image](https://user-images.githubusercontent.com/8447505/31064511-936cf75a-a745-11e7-89af-bacbdbdf7a1d.png)
